ananse/influence.py

```diff
@@ -33,41 +33,165 @@
 Expression = namedtuple("Expression", ["score", "absfc", "realfc"])
 
 
-def read_network(fname, edges=100000):
-    """Read network file and return networkx DiGraph."""
-
-    G = nx.DiGraph()
-
-    rnet = pd.read_csv(fname, sep="\t")
-    rnet.sort_values("prob", ascending=False, inplace=True)
+def read_top_interactions(fname, edges=100000):
+    """Read network file and return the top interactions"""
+    rnet = pd.read_csv(fname, sep="\t")  # read the GRN file
+    rnet.sort_values(
+        "prob", ascending=False, inplace=True
```

Comment:
would it make sense to store the network df sorted instead? If so, you could also use `nrows=edges` in `read_csv`

Reply:
Maybe, but now I'm working on fixing stuf I realize that it might be nice to be able to select which column is used to take the top edges. Sure "prob" should be default, sorting for e.g. the weighted binding I think can also be usefull for quantifying target genes withouth taking expression vallues into account. Therefore sorting it here makes a bit more sense (since you dont have to re-compute the entire network to sort on something else).